sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/ExpressionConverter.java

```diff
@@ -163,6 +164,15 @@
           + " The date_part includes: "
           + INTERVAL_DATE_PART_MSG;
 
+  // Maximum and minimum allowed values for the NUMERIC/DECIMAL data type.
+  // https://github.com/google/zetasql/blob/master/docs/data-types.md#decimal-type
+  private static final BigDecimal MAX_NUMERIC_VALUE =
```

Comment:
This code looks to define the same constants? Can we use those? https://github.com/apache/beam/blob/e9dcd541116098057c23d7132376e60a338adc44/sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlTypesUtils.java#L27

Reply:
That class is in the testing package.

For the record, the same constants are also defined in ZetaSQL Java, but they're private.

https://github.com/google/zetasql/blob/697da738be2af3b6b9492c73c55616d8b9fed0b6/java/com/google/zetasql/Value.java#L102-L105

Reply:
That class is in the test package because it is only used by tests. If it would be useful, you can move it out.
See: https://github.com/apache/beam/pull/12174#discussion_r454618830

Reply:
I moved the constants to ZetaSqlCalciteTranslationUtils.